### PR TITLE
Feature ui

### PR DESCRIPTION
**Pull request by veemish**
kindly please review my PR @BenMaruchu , @chrisoemma ,@Simonmawale.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  
+  /* Responsive layout - when the screen is less than 600px wide, make the two columns stack on top of each other instead of next to each other */
+  @media screen and (max-width: 600px) {
+    .col-25, .col-75, input[type=submit] {
+      width: 100%;
+      margin-top: 0;
+    }
+  }
+ 
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (BenMaruchu) [inline css/main.css]: @veemish  this is supposed to be in your html file

**Review by BenMaruchu — APPROVED** [review]
Hello @veemish 

Nice work,

Couple of things your can improve 
1. You need to format your codes so that they look nice and consistent
2. I see you still need to improve your commit messages those which you did on 11th May were good, but those in 12th May you wrote something like `Doc` , `Feat`, `Document` which is not a recommended way to write commit messages. Please find time to read https://www.conventionalcommits.org/en/v1.0.0-beta.4/

Otherwise nice work and I like how eager you are in learning programming, I hope you will reach far.

:clap: All the best in your next challenge

File: css/main.css (hunk 1 above)
Comment (veemish) [reply css/main.css]: okay, I will have have to review and get it back to you, am not sure why the link is in the CSS file. That's what you mean I hope.@Benmaruchu, I will find out more.

**Comment by veemish** [comment]
> Hello @veemish
> 
> Nice work,
> 
> Couple of things your can improve
> 
> 1. You need to format your codes so that they look nice and consistent
> 2. I see you still need to improve your commit messages those which you did on 11th May were good, but those in 12th May you wrote something like `Doc` , `Feat`, `Document` which is not a recommended way to write commit messages. Please find time to read https://www.conventionalcommits.org/en/v1.0.0-beta.4/
> 
> Otherwise nice work and I like how eager you are in learning programming, I hope you will reach far.
> 
>  All the best in your next challenge




@BenMaruchu  hello, 
thank you for the heads-up and words of encouragement. I will make sure to work on the improvements you mentioned. plus am going to go through the link you also sent thank you!

**Pull request merged**